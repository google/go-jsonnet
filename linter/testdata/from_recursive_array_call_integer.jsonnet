local arr = [0, arr];

[
    arr[0](),
    arr[1][0](),
    arr[1][1][0](),
    arr[1][1][1][1][1][1][1][0](),
]
