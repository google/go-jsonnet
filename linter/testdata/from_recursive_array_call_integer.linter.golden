testdata/from_recursive_array_call_integer:4:5-13 Called value must be a function, but it is assumed to be a number

    arr[0](),


testdata/from_recursive_array_call_integer:5:5-16 Called value must be a function, but it is assumed to be a number

    arr[1][0](),


testdata/from_recursive_array_call_integer:6:5-19 Called value must be a function, but it is assumed to be a number

    arr[1][1][0](),


testdata/from_recursive_array_call_integer:7:5-34 Called value must be a function, but it is assumed to be a number

    arr[1][1][1][1][1][1][1][0](),


